### DISCUSSION TITLE
[Notice] Community Weekly Meeting (June 15th, 2022)

### DISCUSSION THREAD

**Discussion by lijing-21** [apache/apisix#7241] [General]
Hi, community,

Here is the schedule for the coming Weekly Meeting.

Schedule
Date: on Wednesday, June 15th, at 20:00 ~ 22:00 (UTC +8)
Meeting Link:  https://meeting.tencent.com/dm/lq9uKbR6PfmQ

Content:
Apache APISIX is a dynamic, real-time, high-performance open source API gateway that provides rich traffic management features such as load balance, dynamic upstream, authentication, observability, and more. As a traffic portal, the gateway needs to handle an extremely large number of concurrent requests, and for a large gateway, peak QPS can often reach the million level, so performance is a factor that must be focused on for the gateway.

In this meeting, our discussion topic is: **How to do API gateway performance testing right.**

Let's talk about:
- Why do we need performance testing? 
- Why can't we do performance tests on laptops? 
- How can I get the QPS performance for a given CPU load?

The doc of Chinese version:
https://docs.qq.com/doc/DU3Z4bEpTVm5Kc1Fz

Note:
If you have any questions, please comment below or contact [membphis@apache.org](mailto:membphis@apache.org) or
[tzssangglass@apache.org](mailto:tzssangglass@apache.org) .

Kindly let me know if you need Google Meet or Zoom as communication tools.

**Comment by lzxuan** [comment]
Hi, if by any chance the meeting was recorded, may I get a copy?

**Reply by tzssangglass** [reply]
see: https://www.bilibili.com/video/BV1WY4y1g7wd?spm_id_from=333.999.0.0&vd_source=eea59919362e93976fdf64cfd7be4bbb
